lni.cls

```diff
@@ -305,6 +305,21 @@
 \newcommand{\authorrunning}[1]{%
    \fancyhead[LE]{\hspace{0.05cm}\oldsmall\thepage\hspace{5pt}#1}}
 \newcommand*{\email}[1]{\href{mailto:#1}{\urlstyle{same}\protect\nolinkurl{#1}}}
+\newif\iffoot\footfalse
+\let\origthanks\thanks
+\renewcommand{\thanks}[1]{\foottrue\origthanks{#1}\footfalse}
+\newcommand{\orcid}[1]{%
+	\ifstrempty{#1}{}{%
+		\href{https://orcid.org/#1}{
+			\begin{minipage}[t]{\baselineskip}
+				\includegraphics[width=\textwidth]{ORCIDiD_iconvector.pdf}
+			\end{minipage}%
+			\iffoot
+	    		~https://orcid.org/#1%
+			\fi
```

Comment:
I'm not sure of the spaces/tabs before `~`. Thus, I would also add `%` the line before

```suggestion
			\iffoot%
                ~https://orcid.org/#1%
			\fi
```